marlowe-dashboard-client/src/Play/State.purs

```diff
@@ -58,20 +50,18 @@ dummyState = mkInitialState defaultWalletDetails zero $ Minutes zero
     , assets: mempty
     }
 
-mkInitialState :: WalletDetails -> Slot -> Minutes -> State
-mkInitialState walletDetails currentSlot timezoneOffset =
+mkInitialState :: WalletDetails -> Map ContractInstanceId Contract.State -> Minutes -> State
+mkInitialState walletDetails contracts timezoneOffset =
   { walletDetails: walletDetails
   , menuOpen: false
   , screen: ContractsScreen
   , cards: mempty
-  , currentSlot
+  , currentSlot: zero -- this will be updated every second through the websocket
```

Comment:
why do you remove the Slot parameter? I think I added it so that the initial load doesn't show a 300day than later on gets fixed.

Reply:
good point, I think I wasn't thinking 😄  - I'll change it back